From foo@baz Thu Jul 16 12:35:37 PDT 2009
Date: Thu, 16 Jul 2009 12:35:37 -0700
From: Greg Kroah-Hartman <gregkh@suse.de>
Subject: Staging: hv: osd: remove PrintBytes wrapper

From: Greg Kroah-Hartman <gregkh@suse.de>

Use the "real" print_hex_dump_bytes call instead of a wrapper function.

Cc: Hank Janssen <hjanssen@microsoft.com>
Cc: Haiyang Zhang <haiyangz@microsoft.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/staging/hv/ChannelMgmt.c     |    3 ++-
 drivers/staging/hv/StorVsc.c         |    2 +-
 drivers/staging/hv/blkvsc_drv.c      |    5 +++--
 drivers/staging/hv/include/logging.h |   22 ----------------------
 4 files changed, 6 insertions(+), 26 deletions(-)

--- a/drivers/staging/hv/blkvsc_drv.c
+++ b/drivers/staging/hv/blkvsc_drv.c
@@ -578,7 +578,7 @@ static int blkvsc_do_inquiry(struct bloc
 
 	buf = kmap(page_buf);
 
-	//PrintBytes(buf, 64);
+	/* print_hex_dump_bytes("", DUMP_PREFIX_NONE, buf, 64); */
 	// be to le
 	device_type = buf[0] & 0x1F;
 
@@ -603,7 +603,8 @@ static int blkvsc_do_inquiry(struct bloc
 		blkdev->device_id_len = 64;
 
 	memcpy(blkdev->device_id, &buf[8], blkdev->device_id_len);
-	//PrintBytes(blkdev->device_id, blkdev->device_id_len);
+	/* printk_hex_dump_bytes("", DUMP_PREFIX_NONE, blkdev->device_id,
+	 * 			 blkdev->device_id_len); */
 
 	kunmap(page_buf);
 
--- a/drivers/staging/hv/ChannelMgmt.c
+++ b/drivers/staging/hv/ChannelMgmt.c
@@ -687,7 +687,8 @@ VmbusOnChannelMessage(
 	if (hdr->MessageType >= ChannelMessageCount)
 	{
 		DPRINT_ERR(VMBUS, "Received invalid channel message type %d size %d", hdr->MessageType, size);
-		PrintBytes((unsigned char *)msg->u.Payload, size);
+		print_hex_dump_bytes("", DUMP_PREFIX_NONE,
+				     (unsigned char *)msg->u.Payload, size);
 		kfree(msg);
 		return;
 	}
--- a/drivers/staging/hv/include/logging.h
+++ b/drivers/staging/hv/include/logging.h
@@ -117,26 +117,4 @@ extern unsigned int vmbus_loglevel;
 #define DPRINT_EXIT(mod)
 #endif
 
-static inline void PrintBytes(const unsigned char* bytes, int len)
-{
-	int i=0;
-
-	printk("\n<< ");
-	for (i=0; i< len; i++)
-	{
-		printk("0x%x ", bytes[i]);
-	}
-	printk(">>\n");
-}
-
-//
-// Inline
-//
-//static inline void GuidToStr(const GUID g, char *str)
-//{
-//	sprintf(str, "{%02x%02x%02x%02x-%02x%02x-%02x%02x-%02x%02x%02x%02x%02x%02x%02x%02x}",
-//	g[3], g[2], g[1], g[0], g[5], g[4], g[7], g[6], g[8], g[9], g[10], g[11], g[12], g[13], g[14], g[15]);
-//
-//}
-
 #endif //_LOGGING_H_
--- a/drivers/staging/hv/StorVsc.c
+++ b/drivers/staging/hv/StorVsc.c
@@ -713,7 +713,7 @@ StorVscOnIORequest(
 		return -2;
 	}
 
-	//PrintBytes(Request->Cdb, Request->CdbLen);
+	/* print_hex_dump_bytes("", DUMP_PREFIX_NONE, Request->Cdb, Request->CdbLen); */
 
 	requestExtension->Request = Request;
 	requestExtension->Device  = Device;
